compiler/rustc_parse/src/lexer/mod.rs

```diff
@@ -447,17 +451,17 @@ impl<'a> StringReader<'a> {
         &self.src[self.src_index(start)..self.src_index(end)]
     }
 
-    fn report_raw_str_error(&self, start: BytePos, opt_err: Option<RawStrError>) {
-        match opt_err {
-            Some(RawStrError::InvalidStarter { bad_char }) => {
+    fn report_raw_str_error(&self, start: BytePos, prefix_len: u32) {
+        let src = &self.src[start.to_usize() + prefix_len as usize..];
+        let err = rustc_lexer::recompute_raw_string_error(src, prefix_len);
```

Comment:
Is there a reason we can't use inline `recompute_raw_string_error` if it's only used in this place?

Reply:
Because `Cursor` and its impls are private in rustc_lexer, so I made that wrapper.